ops/_main.py

```diff
@@ -445,6 +447,11 @@ def _emit_charm_event(self, event_name: str):
 
         args, kwargs = self._get_event_args(event_to_emit)
         logger.debug('Emitting Juju event %s.', event_name)
+        # If tracing is set up, log the trace id so that tools like jhack can pick it up.
```

Comment:
As @benhoyt suggested, let's add to the comment something about jhack looking for exactly this string, so that people don't accidentally "improve" it in the future.

Reply:
Done, thanks to being alert!